doc/glossary.rst

```diff
@@ -1435,6 +1442,11 @@ functions or non-estimator constructors.
         input into. See :term:`components_` for the special case of affine
         projection.
 
+    ``n_iter_no_change``
+        Number of iterations with no improvement to wait before stopping the
+        iterative procedure. It is typically used with :term:`early stopping` to
+        avoid stopping too early.
```

Comment:
For googlability, we should mention that this parameter is also named "patience" in other libraries.